packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/OverflowMenuDemo/OverflowMenuDemo.tsx

```diff
@@ -167,7 +167,7 @@ export class OverflowMenuDemo extends React.Component {
     });
   };
 
-  onPersistSelect = event => {
+  onPersistSelect = () => {
```

Comment:
As a demo, it would be good to show what args can be are expected here

Reply:
Added argument back in